src/backend/api/meals.js

```diff
@@ -12,4 +15,65 @@ router.get("/", async (request, response) => {
   }
 });
 
+
+router.get("/future-meals", async (request, response) => {
+  try {
+    // knex syntax for selecting things. Look up the documentation for knex for further info
+    const futureMeals = await knex("meals").select(["id", "title", "description", "price"]).where("meal_when", ">", formattedCurrentDate);
+    response.json(futureMeals);
+  } catch (error) {
+    throw error;
```

Comment:
Have you tested what the client receives in case of error?
Perhaps consider sending a relevant HTTP status code back to the client. The aim here would be to make sure the user is guided in the correct way once an issue occurs.

Reply:
Do you mean something like this?
catch (error) {
    console.error(error);

    response.status(500).json({ error: "An unexpected error occurred while processing your request." });
  }